skimage/exposure/hdr.py

```diff
@@ -0,0 +1,311 @@
+import numpy as np
+
+
+def get_radiance(images, exposure, radiance_map):
+    """
+    Return the radiance image for a series of images based upon a camera
+    response function.
+
+    Parameters
+    ----------
+    images: list
+        List of images in the for of numpy arrays. Either mono or color
+        (RGB, MxNx3).
+    exposure : numpy 1D array
+        Array of exposure times in seconds.
+    radiance_map : numpy array
+        Array mapping the counts to radiance
+
+    Returns
+    -------
+    hdr : numpy array
+        Resulting image with radiance values.
+    """
+
+    den = np.ones(images[0].shape)
+    num = np.zeros(images[0].shape)
+    wf = np.vectorize(_weight_func)
+    for idx, im in enumerate(images):
+        gij = im.copy()
+        # For colour
+        if im.ndim == 3:
+            for ii in range(im.shape[2]):
+                gij[:, :, ii] = radiance_map[im[:, :, ii] + 1, ii]
```

Comment:
@jni may have thoughts on these lines

Reply:
@jni is just confused. 😅 I mean, at least the line below in the else clause seems incorrect — there is no ii defined there? Anyway I don't understand what a radiance map is — what is the shape and what are the values?

Reply:
I agree that the ii in the line below is incorrect, I'll correct that.
I'll go to and add comments as well, so that it becomes more readable as this is slightly complicated.